### ISSUE TITLE
Recursive population in queries

### ISSUE THREAD

**Issue by alnorth** [balderdashy/waterline#1157]
Issue #308 was closed prematurely. It's no use saying we should subscribe to an issue for updates and then just closing the issue.

**Comment by particlebanana** [comment]
We have https://github.com/balderdashy/waterline/pull/1052 that is almost ready to be merged.

**Comment by atiertant** [comment]
@particlebanana waiting your PR on my repo

**Comment by tcury** [comment]
+1

**Comment by chacliff** [comment]
do they have any time estimates on #1052 it would be really nice to have this. I'm using hapi-waterline, but i'm hoping just by updating its supporting waterline once this is merged with the master branch that it will work out of the box.

**Comment by sailsbot** [comment]
Thanks for posting, @alnorth.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
